src/main/java/ttt/gui/GuiPrompt.java

```diff
@@ -0,0 +1,64 @@
+package ttt.gui;
+
+import ttt.GameType;
+import ttt.board.Board;
+import ttt.player.PlayerSymbol;
+
+public class GuiPrompt implements GameRulesPrompt {
+    private BoardPresenter boardPresenter;
+    private Board board;
+    private PlayerSymbol currentPlayer;
+
+    public GuiPrompt(BoardPresenter boardPresenter) {
+        this.boardPresenter = boardPresenter;
+        this.currentPlayer = PlayerSymbol.X;
+    }
+
+    GuiPrompt(BoardPresenter boardPresenter, Board board) {
+        this.boardPresenter = boardPresenter;
```

Comment:
This constructor is only used by the tests.  The board dependency is set via GuiPrompt::print(Board board) method.  This could cause problems if for some reason you didn't call ::print(..) before ::playMoveAt